ftps/1.6/modules/ROOT/pages/ftps-troubleshooting.adoc

```diff
@@ -0,0 +1,39 @@
+= Troubleshoot FTPS Connector - Mule 4
+
+To troubleshoot Anypoint Connector for FTPs (FTPS Connector), become familiar with the information about troubleshooting Mule application issues
+
+== Troubleshoot Mule Application Issues
+
+Sometimes you are not sure why the application is failing and you require a better understanding of what is going on in the app's interaction with the FTPS server.
+
+To resolve this error:
+
+* Use the configuration file to increase the logging level on the server side.
+
+* Try to replicate the issue using an FTPS client, for example, Cyberduck or Filezilla. If the issue also occurs with these clients, then it is likely that the issue is not related to the Mule app.
+
+* Get better and more detailed logs on the Mule app side by enabling verbose logging for FTPS Connector and for the JSCH driver:
++
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the `src/main/resources` path folder.
+. Open the `log4j2.xml` file inside the folder.
+. If the following line is already in the `log4j2.xml` file, uncomment it to enable it; otherwise, add it:
++
+[source,xml,linenums]
+----
+    <AsyncLogger name=“org.mule.extension.ftps” level=“TRACE”/>
+    <AsyncLogger name=“com.jcraft.jsch” level=“DEBUG”/>
```

Comment:
jsch library is not used on the FTPS connector. Instead, apache.commons.net is used through its dependency on the FTP connector
`<AsyncLogger name=“org.mule.extension.ftp” level=“TRACE”/>`
`<AsyncLogger name=“org.apache.commons.net” level=“DEBUG”/>`
can be added to the logging.